phone/src/apps/messages/components/modal/MessageBubble.tsx

```diff
@@ -0,0 +1,75 @@
+import { Box, makeStyles, Paper, Typography } from '@material-ui/core';
+import React from 'react';
+import { Message } from '../../../../common/typings/messages';
+
+const useStyles = makeStyles((theme) => ({
+  imageMessage: {
+    width: '80%',
+    maxWidth: '80%',
+    height: '80%',
+    maxHeight: '80%',
+  },
+  mySms: {
+    float: 'right',
+    margin: theme.spacing(1),
+    padding: '6px 12px',
+    height: 'auto',
+    width: '80%',
```

Comment:
Let's set a min and max width on the papers, so the message 'bubble' doesn't stretch so much as it does now,